uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-request-popup-details-tab.component.spec.js

```diff
@@ -2093,9 +1588,10 @@ define([
      */
     function compileComponent (params) {
```

Comment:
why isn't this function shared in the request modal helpers?

Reply:
Because it needs access of `controller` variable which is defined in the spec files. So we would need to return that as well with params. We can still do it by returning an object containing both params and controller, but it would be too complicated and not worth the time IMO.